com.tux4kids.tuxtype.json

```diff
@@ -40,6 +40,10 @@
         {
             "name": "t4kcommon",
             "rm-configure": true,
+            "build-options": {
+                "cflags": "-fcommon",
+                "cxxflags": "-fcommon"
```

Comment:
Wow, I wish I'd known about this flag when I spent hours trying to fix these problems in t4kcommon!